src/app/components/mno-app-install-btn/mno-app-install-btn.coffee

```diff
@@ -167,8 +167,7 @@ angular.module 'mnoEnterpriseAngular'
             products = response.products?.products
             currency = MnoeOrganizations.selected.organization.billing_currency || MnoeConfig.marketplaceCurrency()
             plans = vm.app.pricing_plans
-            if !plans[currency]
-              vm.arePlansAvailable = false
+            vm.arePlansAvailable = !plans[currency] || !ProvisioningHelper.pricedPlan(plans.default[0])
```

Comment:
Seems like this logic is reversed.

`vm.arePlansAvailable = plans[currency] || ...`

i.e. plans are available if the plans have our currency.

Also why are we only checking the first default plan -- there are products with multiple default plans.

Reply:
We also need to account for times where product pricings are skipped for products.

Reply:
```coffeescript
# TODO: Comments
vm.orderingPossible = plans[currency] || skipPricing || !ProvisioningHelper.hasPricedPlan(plans.default)
```